applications/GeoMechanicsApplication/tests/cpp_tests/input_utility_stub.cpp

```diff
@@ -0,0 +1,49 @@
+// KRATOS___
+//     //   ) )
+//    //         ___      ___
+//   //  ____  //___) ) //   ) )
+//  //    / / //       //   / /
+// ((____/ / ((____   ((___/ /  MECHANICS
+//
+//  License:         geo_mechanics_application/license.txt
+//
+//  Main authors:    Richard Faasse
+//
+
+
+#include "input_utility_stub.h"
+
+namespace Kratos {
+
+InputUtilityStub::InputUtilityStub(const std::string& rParameterJsonString) :
+    mParameterJsonString{rParameterJsonString}
+{
+
+}
+
+Parameters InputUtilityStub::ProjectParametersFrom(const std::string& rProjectFilePath) const
+{
+    Parameters result(mParameterJsonString);
+
+    return result;
+}
+
+void InputUtilityStub::ReadModelFromFile(const std::filesystem::path& rModelPartFilePath, ModelPart& rModelPart) const
+{
+    mNumberOfReadCalls++;
```

Comment:
Prefer pre-increment operators over post-increment operators (the latter ones are more efficient), so:
```suggestion
    ++mNumberOfReadCalls;
```

Reply:
Done